data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ITable.java

```diff
@@ -218,4 +222,70 @@ default boolean isEmpty() {
     default int[] getShape() {
```

Comment:
Doesn't understand this method

Reply:
I suppose return the "shape" of the table (a list containing the number of columns and the number of rows) ?

Reply:
Is it possible to find a more descriptive name ?

Reply:
Yes, this method return the size of each dimension of the IDataSet. For an ITable, it return an array with the row and column count.
For the name, I'm open to any idea !

Reply:
getSize ?

Reply:
ok